generator/tests/builtin-extensions/builtin-extensions

```diff
@@ -0,0 +1,52 @@
+@theia/vscode-builtin-bat@0.2.1
+@theia/vscode-builtin-clojure@0.2.1
+@theia/vscode-builtin-coffeescript@0.2.1
+@theia/vscode-builtin-configuration-editing@0.2.1
+@theia/vscode-builtin-cpp@0.2.1
+@theia/vscode-builtin-csharp@0.2.1
+@theia/vscode-builtin-css@0.2.1
+@theia/vscode-builtin-debug-auto-launch@0.2.1
+@theia/vscode-builtin-docker@0.2.1
+@theia/vscode-builtin-emmet@0.2.1
+@theia/vscode-builtin-fsharp@0.2.1
+@theia/vscode-builtin-git@0.2.1
+@theia/vscode-builtin-git-ui@latest
```

Comment:
here it's latest ?

Reply:
I will remove it until stable version appear.